health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -1,76 +1,20 @@
-import { useLocation, useHistory, Link, useParams } from "react-router-dom";
-import React, { useState, Fragment } from "react";
-import { DeleteIconv2, DownloadIcon, FileIcon, Button, Card, CardSubHeader, EditIcon, ArrowForward } from "@egovernments/digit-ui-react-components";
-import { Button as ButtonNew, Dropdown } from "@egovernments/digit-ui-components";
-
 //create functions here based on module name set in mdms(eg->SearchProjectConfig)
 //how to call these -> Digit?.Customizations?.[masterName]?.[moduleName]
 // these functions will act as middlewares
-// var Digit = window.Digit || {};
-
-
+var Digit = window.Digit || {};
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)